mission_control/include/mission_control/behavior.h

```diff
@@ -66,7 +65,7 @@ class Behavior : public BT::AsyncActionNode
         setStatus(behaviorRunningProcess());
         break;
       case BT::NodeStatus::SUCCESS:
-        halt();
+        setStatus(BT::NodeStatus::SUCCESS);
```

Comment:
@patapufate isn't this redundant? Same for the `case` below. I did not noticed before.

Reply:
Yes. You're right. Now that isn't AsyncActionNode Type, is redundant

Reply:
My point is, why setting status to the same value it already has? Code won't reach this line until `ActionNodeBase::status()` returns `BT::NodeStatus::SUCCESS`, so why setting it again to the same value?

Reply:
Moreover, it looks like `TreeNode::executeTick()` already sets the status (see [here](https://github.com/BehaviorTree/BehaviorTree.CPP/blob/f54f6d83e5c06f44eae2b4d9700f5178dbdb4159/src/tree_node.cpp#L36)). 

Perhaps we don't even need to set status, only:

- On IDLE or RUNNING, return `behaviorRunningProcess()` outcome
- On SUCCESS or FAILURE, return the current `status()`

Reply:
In which case we may want to give a reasonable name to the class, something that conveys this one-off behavior.